filters/flowid/hashing.go

```diff
@@ -10,33 +10,53 @@ import (
 const (
 	flowIdAlphabet  = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-+"
 	alphabetBitMask = 63
-)
-
-const (
-	MaxLength = 64
-	MinLength = 8
+	MaxLength       = 64
+	MinLength       = 8
+	defaultLen      = 16
 )
 
 var (
 	ErrInvalidLen = errors.New(fmt.Sprintf("Invalid length. Must be between %d and %d", MinLength, MaxLength))
 	flowIdRegex   = regexp.MustCompile(`^[0-9a-zA-Z+-]+$`)
 )
 
-// NewFlowId returns a random flowId using the flowIdAlphabet with length l.
-//
+// NewFlowId creates a new built-in generator with the defined length and returns a flowid
+// This exported function is deprecated and the new flowIdGenerator interface should be used
```

Comment:
Was trying to avoid breaking backwards compatibility. I saw that we ended up using it also to generate random route ids.